docs/source/getting-started.rst

```diff
@@ -38,7 +38,7 @@ Download StreamAlert
 
 .. code-block:: bash
 
-  git clone --branch stable https://github.com/airbnb/streamalert.git
+  git clone --branch release-3-0-0 https://github.com/airbnb/streamalert.git
```

Comment:
We always recommend to checkout `stable` branch. The release branch is for development only and it will be tagged as stable when we cut the release.

Reply:
ok, let me role this back :)